src/workers/parser/utils/helpers.js

```diff
@@ -67,3 +67,10 @@ export function getMemberExpression(root: Node) {
   const expr = _getMemberExpression(root, []);
   return expr.join(".");
 }
+
+export function isJSXElement(path: NodePath): boolean {
+  if (t.isJSXElement(path)) {
+    return true;
+  }
+  return false;
+}
```

Comment:
we can simplify this to `return t.isJSXElement(path)` at which point i am not sure if we need to have a helper function :)